_sass/module/_hover-info.scss

```diff
@@ -0,0 +1,27 @@
+.hover-info {
+  visibility: hidden;
+// Jesse asks: I tried tinkering with _typography.scss to create a "$font-size-small" variable/setting but failed. What should we do to accomodate the couple spots where a really small font is present in designs? //
```

Comment:
We could create a new variable like `font-size-xsmall` or something. Or, if this xsmall text is more of a component, the font size could be defined as part of the component.